src/lib.rs

```diff
@@ -71,6 +71,9 @@ mod storage;
 pub mod filter;
 pub use filter::{Bloom, BloomDataProvider, BloomProvider, Config as BloomConfig, FilterResult};
 
+/// tools to interact with pearl structures
+pub mod tools;
```

Comment:
I think you can name this module more specific, like `recovery` or `reconstruction`

Reply:
But this is also `migration`

Reply:
Yes, you're right... Then I guess we have to use generic name

Reply:
Maybe `blob_tools` or `file_tools`